app/src/main/java/org/cru/godtools/fragment/ToolDetailsFragment.java

```diff
@@ -280,6 +289,20 @@ private void updateViews() {
             mTitle.setText(ModelUtils.getTranslationName(getContext(), mLatestTranslation, mTool));
         }
         bindShares(mShares, mTool);
+        if (mBanner != null) {
+            mBanner.asImageView().setVisibility(
+                    (mTool != null && TextUtils.isEmpty(mTool.getOverviewVideo())) ||
+                            Build.VERSION.SDK_INT < Build.VERSION_CODES.JELLY_BEAN_MR1 ?
+                            View.VISIBLE : View.GONE);
+        }
+        if (mVideoBanner != null) {
+            mVideoBanner.setVisibility(
+                    mTool != null && !TextUtils.isEmpty(mTool.getOverviewVideo()) ? View.VISIBLE :
```

Comment:
This conditional logic is here twice, would it be better to stick the `setVisibility` call inside the if block below, then have an else setting it to `View.GONE`?

Reply:
Also, would we want to check the SDK version before setting visibility?

Reply:
> Also, would we want to check the SDK version before setting visibility?

We would want to check the version because there may be an instance where we have a video but the player is not able to be played. (The current video implementation doesn't work on lower versions of Android

Reply:
Ok, so that should probably be added to this conditional then.

Reply:
@wrandall22 `mVideoBanner` will be null on Older devices.

Reply:
I still think the conditional could be simplified to something like:
```
if (mVideoBanner != null) {
  if (mTool != null && !TextUtils.isEmpty(mTool.getOverviewVideo())) {
    mVideoBanner.setVisibility(View.VISIBLE);
    updateOverviewVideo();
  } else {
    mVideoBanner.setVisibility(View.GONE);
  }
}
```